jwql/website/apps/jwql/data_containers.py

```diff
@@ -65,6 +68,36 @@
 REPO_DIR = os.path.split(PACKAGE_DIR)[0]
 
 
+def get_table_view_components(request):
+    """Render table view.
```

Comment:
Please include a `Parameters` section here that describes what `request` is.

Reply:
Got it, change is made. I added

```
Returns
--------
None
```

Just to be explicit.